public/src/index.js

```diff
@@ -16,22 +16,46 @@ import EditAccountInfo from './components/edit_account';
 
 import { authTransition } from './actions/login';
 
-const Root = props => {
-  //isLogged in will go here, does not exist yet!
-  //login redirect if logged in is true needs to redirect to the landing page
+const Root = () => {
+  // checks if user id !==null and if authToken exists iin localStorage;
+  // will refactor to seperate concerns;
   const isLoggedIn = authTransition.bind(null, store);
```

Comment:
This is binding the authTransition function to always call with store as one of it's arguments. We can go over .bind, .call, .apply some time next week in more depth if you're interested. :smile: